static/js/browser2/arc_section.tsx

```diff
@@ -97,8 +98,13 @@ export class ArcSection extends React.Component<
           inLabels: labelsData["inLabels"],
           outLabels: labelsData["outLabels"],
           provDomain,
+          dataFetched: true,
         });
-      }
-    );
+      })
+      .catch(() => {
+        this.setState({
+          dataFetched: true,
```

Comment:
i suppose it's whether the fetch attempt has completed, not that the data has been fetched. should an error message be displayed?

Reply:
done

Reply:
done